### ISSUE TITLE
Reference to USD should be replaced with BTC or removed completely.

### ISSUE THREAD

**Issue by qtipwax** [LiskHQ/lisk-ui#44]
I would like the reference to USD removed.  It is biased towards one country and cypto should try to move away from fiat references when possible.  Possible solutions.

1)  Replaced with BTC value would be more appropriate in my opinion.
2)  If not all languages that Lisk supports should have their currency listed.  
3) References to other fiat or crypto removed completely.

**Comment by qtipwax** [comment]
<img width="646" alt="usd" src="https://cloud.githubusercontent.com/assets/12992198/15227288/50f0fcae-183b-11e6-87e5-4fa766b87f06.PNG">

**Comment by densmirnov** [comment]
I disagree. First of all, the ability to know the value of your assets without accessing the exchange is very handy. Second, USD is like universal standard for crypto accepted by everyone no matter what country are you from. Just as Bitcoin. Rather, I would add an option to select which currency you prefer to see. But in any case, it wouldn't remove it.

**Comment by mrv777** [comment]
Looks like you could change:
``` 
$scope.convertToUSD = function (lisk) {
        return (lisk / 100000000) * $scope.lisk_usd;
    };
 ```

to something like this to get the new value:
```
$scope.convertToUSD = function (lisk) {
        if (someSetting=='btc')
                return (lisk / 100000000) * $scope.lisk_btc;
        else
                return (lisk / 100000000) * $scope.lisk_usd;
    };
```

Would also need to update these with an if statement to make it obvious it is BTC and not USD now:
partials/modals/userInfo.html 
partials/modals/transactionInfo.html 
partials/modals/blockInfo.html

**Comment by qtipwax** [comment]
My issue with USD is that I need to convert it again into the "local" currency whether Chinese Yuan, Korean Won, Euro, Krone, Rupee, etc.  If using BTC there is only one conversion and I know the value.

**Comment by fix** [comment]
Closing it as won't fix, what do you think @karmacoma ?
